packages/apollo-mst/src/AnnotationFeatureModel.ts

```diff
@@ -240,6 +240,48 @@ export const AnnotationFeatureModel = types
         transcript.filter((transcriptPart) => transcriptPart.type === 'CDS'),
       )
     },
+    get looksLikeGene(): boolean {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any
+      const session = getSession(self) as any
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      const { apolloDataStore } = session
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
+      const { featureTypeOntology } = apolloDataStore.ontologyManager
+      if (!featureTypeOntology) {
+        return false
+      }
+      const children = self.children as Children
+      if (!children?.size) {
+        return false
+      }
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      const isGene =
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+        featureTypeOntology.isTypeOf(self.type, 'gene') ||
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+        featureTypeOntology.isTypeOf(self.type, 'pseudogene')
+      if (!isGene) {
+        return false
+      }
+      for (const [, child] of children) {
+        if (
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+          featureTypeOntology.isTypeOf(child.type, 'transcript') ||
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+          featureTypeOntology.isTypeOf(child.type, 'pseudogenic_transcript')
+        ) {
+          const { children: grandChildren } = child
+          if (!grandChildren?.size) {
+            return false
+          }
+          return [...grandChildren.values()].some((grandchild) =>
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+            featureTypeOntology.isTypeOf(grandchild.type, 'exon'),
```

Comment:
One slight concern I have here, which was introduced in [this](https://github.com/GMOD/Apollo3/commit/72def4bef098f59907390dee147e1761b91eee18) commit, is that a gene feature with CDS but no exon features will now not be regarded as a gene. Was that a deliberate choice, or oversight?